modules/cloud-credential-operator.adoc

```diff
@@ -8,8 +8,274 @@
+actions that require additional credentials or permissions, such as upgrading,
+to proceed.
+
+The requirement that mint mode stores the admin-level credential in the cluster
+`kube-system` namespace might not suit the security requirements of every
+organization.
+
+When using the CCO in mint mode, ensure that the credential you provide meets
+the requirements of the cloud on which you are running or installing
+{product-title}. If the provided credentials are not sufficient for mint mode,
+the CCO cannot create an IAM user.
+
+.Mint mode credential requirements
+[cols="1,9a"]
+|====
+|Cloud |Permissions
+
+|AWS
+|* `iam:CreateAccessKey`
+* `iam:CreateUser`
+* `iam:DeleteAccessKey`
+* `iam:DeleteUser`
+* `iam:DeleteUserPolicy`
+* `iam:GetUser`
+* `iam:GetUserPolicy`
+* `iam:ListAccessKeys`
+* `iam:PutUserPolicy`
+* `iam:TagUser`
+* `iam:SimulatePrincipalPolicy`
+
+|Azure
+|Service principal with the permissions specified in the xref:../installing/installing_azure/installing-azure-account.adoc#installation-azure-service-principal_installing-azure-account[Creating a service principal]
+section of the xref:../installing/installing_azure/installing-azure-account.adoc[Configuring an Azure account]
+page.
+
+|GCP
+|* `resourcemanager.projects.get`
+* `serviceusage.services.list`
+* `iam.serviceAccountKeys.create`
+* `iam.serviceAccountKeys.delete`
+* `iam.serviceAccounts.create`
+* `iam.serviceAccounts.delete`
+* `iam.serviceAccounts.get`
+* `iam.roles.get`
+* `resourcemanager.projects.getIamPolicy`
+* `resourcemanager.projects.setIamPolicy`
+
+|====
+
+[discrete]
+===== Mint mode with removal or rotation of the admin-level credential
+Mint mode with removal or rotation of the admin-level credential is supported
+for AWS in {product-title} version 4.4 and later.
+
+This option requires the presence of the admin-level credential during
+installation, but the credential is not stored in the cluster permanently and
+does not need to be long-lived.
+
+After installing {product-title} in mint mode, you can remove the admin-level
+credential Secret from the cluster. If you remove the Secret, the CCO uses a
+previously minted read-only credential that allows it to verify whether all
+`credentialsRequest` CRs have their required permissions. Once removed, the
+associated credential can be destroyed on the underlying cloud if desired.
+
+The admin-level credential is not required unless something that requires an
+admin-level credential needs to be changed, for instance during an upgrade.
+Prior to each upgrade, you must reinstate the credential Secret with the
+admin-level credential. If the credential is not present, the upgrade might be
+blocked.
+
+[discrete]
+==== Passthrough mode
+Passthrough mode is supported for AWS, Azure, GCP, {rh-openstack-first},
+{rh-virtualization-first}, and VMware vSphere.
+
+In passthrough mode, the CCO passes the provided non-admin cloud credential to
```

Comment:
Don't know if you want to include this level of detail, but just for clarification: the text here calls it a non-admin credential, but CCO doesn't care if it is or isn't admin (when you have explicitly opted into credentialsMode: Passthrough). CCO will take the credentials as-is and give them to other components even if the creds are admin-level. CCO will not try to create new credentials in passthrough mode even if technically the creds are capable of supporting this.

Reply:
ah ok. so they can use a mint-capable cred for passthrough, but don't have to. Will rephrase a bit.